cardano-cli/src/Cardano/CLI/EraBased/Run.hs

```diff
@@ -22,72 +21,85 @@ import           Cardano.CLI.EraBased.Vote
+  => EraBasedCommand era -> ExceptT AnyEraCmdError IO ()
 runEraBasedCommand = \case
   EraBasedGovernanceCmds cmd -> runEraBasedGovernanceCmds cmd
 
 runEraBasedGovernanceCmds :: ()
   => EraBasedGovernanceCmds era
-  -> ExceptT () IO ()
+  -> ExceptT AnyEraCmdError IO ()
 runEraBasedGovernanceCmds = \case
   EraBasedGovernancePreConwayCmd w ->
     runEraBasedGovernancePreConwayCmd w
   EraBasedGovernancePostConwayCmd w ->
     runEraBasedGovernancePostConwayCmd w
   EraBasedGovernanceMIRPayStakeAddressesCertificate w mirpot vKeys rewards out ->
-    firstExceptT (const ()) -- TODO: Conway era - fix error handling
-      $ runGovernanceMIRCertificatePayStakeAddrs w mirpot vKeys rewards out
+    runGovernanceMIRCertificatePayStakeAddrs w mirpot vKeys rewards out
+      & firstExceptT AnyEraCmdGovernanceCmdError
 
   EraBasedGovernanceMIRTransfer w ll oFp direction ->
-    firstExceptT (const ()) -- TODO: Conway era - fix error handling
-      $ runGovernanceMIRCertificateTransfer w ll oFp direction
+    runGovernanceMIRCertificateTransfer w ll oFp direction
+      & firstExceptT AnyEraCmdGovernanceCmdError
 
   EraBasedGovernanceDelegationCertificateCmd stakeIdentifier delegationTarget outFp ->
-    firstExceptT (const ()) -- TODO fix error handling
-      $ runGovernanceDelegationCertificate stakeIdentifier delegationTarget outFp
+    runGovernanceDelegationCertificate stakeIdentifier delegationTarget outFp
+      & firstExceptT AnyEraCmdEraDelegationError
 
   EraBasedGovernanceRegistrationCertificateCmd regTarget outFp ->
-    firstExceptT (const ()) -- TODO: Conway era - fix error handling
-      $ runGovernanceRegistrationCertificate regTarget outFp
+    runGovernanceRegistrationCertificate regTarget outFp
+      & firstExceptT AnyEraCmdEraBasedRegistrationError
+
   EraBasedGovernanceVoteCmd anyVote outFp ->
-    firstExceptT (const ()) -- TODO: Conway era - fix error handling
-      $ runGovernanceVote anyVote outFp
+    runGovernanceVote anyVote outFp
+      & firstExceptT AnyEraCmdEraBasedVoteError
 
   EraBasedGovernanceCommitteeCmds cmds ->
-    firstExceptT (const ()) -- TODO: Conway era - fix error handling
-      $ runGovernanceCommitteeCmds cmds
+    runGovernanceCommitteeCmds cmds
+      & firstExceptT AnyEraCmdGovernanceCommitteeError
 
   EraBasedGovernanceActionCmds cmds ->
-    firstExceptT (const ()) -- TODO: Conway era - fix error handling
-      $ runGovernanceActionCmds cmds
+    runGovernanceActionCmds cmds
+      & firstExceptT AnyEraCmdGovernanceActionError
 
   EraBasedGovernanceDRepGenerateKey w vrf sgn ->
-    firstExceptT (const ()) -- TODO: Conway era - fix error handling
-      $ runGovernanceDRepKeyGen w vrf sgn
+    runGovernanceDRepKeyGen w vrf sgn
+      & firstExceptT AnyEraCmdGovernanceCmdError
+
 
 runEraBasedGovernancePreConwayCmd
   :: ShelleyToBabbageEra era
-  -> ExceptT () IO ()
+  -> ExceptT e IO ()
 runEraBasedGovernancePreConwayCmd _w = pure ()
 
 runEraBasedGovernancePostConwayCmd
```

Comment:
I think we can actually remove `runEraBasedGovernancePostConwayCmd` and `runEraBasedGovernancePreConwayCmd`

Reply:
This got addressed in https://github.com/input-output-hk/cardano-cli/pull/180